Source/PowerMeter/PowerMeter.cs

```diff
@@ -41,7 +41,7 @@ public struct CommonConfiguration
             public double Trigger_Level;
 
             /// <summary>
-            /// Specifies the filter function of a sensor on or off.
+            /// Specifies whether auto-averaging mode is enabled or not.
             /// </summary>
             public bool Averaging;
```

Comment:
The description here is clarifying, which is good. However, I think that the variable name is still deceiving. Our other functions have an "Averaging Enabled" boolean to indicate whether any averaging should occur. This should be named something like "AutoAveragingEnabled" to clarify the intended use.

Reply:
Renamed it to "Averaging_Auto_Enabled" to be consistent with the parameter name of  the ConfigureAveragingAutoEnabled function.